src/main/scala/nl/tradecloud/kafka/KafkaSubscriber.scala

```diff
@@ -25,7 +25,7 @@ class KafkaSubscriber(
     batchingSize: Int = 1,
     batchingInterval: FiniteDuration = 3.seconds,
     system: ActorSystem,
-    offset: ConsumerOffset.Value
+    initialOffset: Option[ConsumerOffset.Value] = None
```

Comment:
rename initialOffset to autoResetConfig, equivalent to Kafka config, initialOffset is not a better name, autoResetConfig can be found in Kafka docs

Reply:
rename ConsumerOffset to AutoResetConfig, with values Latest, Earliest